src/style.css

```diff
@@ -50,17 +74,145 @@ footer {
 footer p {
   color: white;
   padding: 20px;
+  font-size: 25px;
+  text-align: center;
+  font-family: monospace;
 }
 
 .cards {
   display: flex;
-  justify-content: space-evenly;
+  justify-content: center;
   flex-wrap: wrap;
+  color: white;
+  text-align: center;
+  padding-top: 30px;
+  overflow-y: scroll;
+}
+
+.parent {
+  display: flex;
+  justify-content: center;
+}
+
+.commentText {
+  width: 400px;
+  height: 200px;
+}
+
+.commentArea {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 60%;
+}
+
+#name {
+  padding: 15px;
+  color: white;
+  background-color: rgb(53, 14, 14);
+  border-radius: 5px;
+  border: none;
+  outline: none;
+}
```

Comment:
- [ OPTIONAL] There are similar CSS rule name which name "name" it's can be created different name 

![image](https://user-images.githubusercontent.com/24355301/175649825-65c4cd30-d2a8-4900-8ad1-86378473b916.png)